src/main/groovy/com/marklogic/gradle/task/roxy/CopyRoxyFilesTask.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task.roxy
+
+import com.marklogic.gradle.task.MarkLogicTask
+import org.apache.commons.io.FileUtils
+import org.gradle.api.tasks.TaskAction
+
+import static groovy.io.FileType.FILES
+
+class CopyRoxyFilesTask extends MarkLogicTask {
+
+	def roxyFolderMapping = [
```

Comment:
MarkLogicTask has a getAppConfig method, which returns an AppConfig object. This has a "List<String> getModulePaths" method, which returns a list of all the paths to modules that the developer has configured. Normally there is only one, and normally the path is the default - i.e. "src/main/ml-modules". But to avoid any potential issues, this mapping should call getModulePaths and use the first string that's returned. 

The mapping here can still be hardcoded, but the "src/main/ml-modules" should be removed from each entry. And then when the mapping is used, the first string from getModulePaths can be prepended to the mapped value.